interfaces/inc/c_pal/gballoc_hl.h

```diff
@@ -11,6 +11,8 @@
 #include <stdint.h>
 #endif
 
+#include "malloc_multi_flex.h"

```

Comment:
I think malloc_multi_flex is built on top of malloc if we want to not copy/pasta it in all the gballoc implementations.

Arguably malloc_flex and malloc_2 is actually the same, but those are already in place there in the gballoc API.